signup/signup.js

```diff
@@ -372,16 +134,17 @@ function updateSitesDatesAndTimes() {
 		$("#timePicker").show();
 		// Clear any previously selected date
 		$("#timePickerSelect").html('');
-		var value = this.value;
+		var site = this.value;
 		var date = $("#dateInput").val();
 
-		let siteObj = dateSitesTimes.getDate(new Date(date)).getSite(value);
+		dateSitesTimes.updateGlobalTimes(date, site);
 		var timeSelect = $("#timePicker select")
 		timeSelect.append($('<option disabled selected value="" style="display:none"> -- select an option -- </option>'));
-		for(const time of siteObj.getTimesArray()) {
+		for(const time in times) {
 			timeSelect.append($('<option>', {
 				value: time,
-				text : time
+				text : time,
+				disabled: times[time] > 0 ? false : true
```

Comment:
Similarly here, should we disable it or just now show it? If we're disabling it, maybe we say something like "FULL" next to it so people know?